openssl/src/cms.rs

```diff
@@ -219,11 +219,61 @@ impl CmsContentInfo {
             Ok(CmsContentInfo::from_ptr(cms))
         }
     }
+
+    /// Verify this CmsContentInfo's signature, given a stack of certificates
+    /// in certs, an X509 store in store. If the signature is detached, the
+    /// data can be passed in data. The data sans signature will be copied
+    /// into output_data if it is present.
+    ///
+    /// OpenSSL documentation at [`CMS_verify`]
+    ///
+    /// [`CMS_verify`]: https://www.openssl.org/docs/manmaster/man3/CMS_verify.html
+    pub fn verify(
+        &mut self,
+        certs: Option<&Stack<X509>>,
```

Comment:
This should take a `&StackRef`, not `&Stack`.

Reply:
Changed to &StackRef.

Reply:
Did a commit not get pushed? I still see `&Stack<X509>` here.

Reply:
Sorry. Editing error.